bin/export_sc_atlas_ebeye_xml.pl

```diff
@@ -124,7 +125,12 @@ sub check_env_var {
 
 add_entry_count ( $configHash );
 
+sub LWP::UserAgent::get_basic_credentials {
+     my ($self, $realm, $url, $isproxy) = @_;
 
+     return $ENV{'SOLR_USER'}, $ENV{'SOLR_PASS'};
+}
```

Comment:
Would it benefit wrapping this method [to turn off the redefine warning](https://perlmaven.com/lwp-useragent-and-basic-authentication)?
```
{
    no warnings 'redefine';
...
}

Reply:
It may be worth keeping the warning up for future generations..